### PR TITLE
Suggested changes after review.

### PR DESCRIPTION
**Pull request by MartinEden**

### CODE REVIEW

File: check.py [hunk 1]
```diff
@@ -52,10 +52,11 @@ def check_table(table_name):
...
     new_rows = get_table(table_name, database_settings["new_db_name"])
     for k in original_rows.keys():
-        if not original_rows[k] == new_rows[k]:
-            # only do this more expensive equality check if the first one fails
-            if not utils.ARRAYEQUAL(original_rows[k], new_rows[k]):
-                print("Change in row {rowid}: was {old}, now {new}".format(rowid=k, old=original_rows[k], new=new_rows[k]))
+        original = original_rows[k]
+        new = new_rows[k]
+
+        if original[k] != new[k] and (not utils.array_equal(original, new)):
```
Comment (MartinEden) [inline check.py:58]: As in most languages, boolean logic is lazy - the second more expensive half will only be evaluated if the first half is true.

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+psycopg2
```
Comment (MartinEden) [inline requirements.txt:1]: It's good practice in Python projects to include any dependent packages in a requirements.txt file with each package on a separate line. Then they can be installed using `pip3 install -r requirements.txt`.

File: utils.py [hunk 3]
```diff
@@ -1,11 +1,11 @@
 
-def EQUAL(a, b):
+def sql_equal(a, b):
     return a == b or ((a != a) and (b != b))
 
 
-def ARRAYEQUAL(x, y):
+def array_equal(x, y):
```
Comment (MartinEden) [inline utils.py:6]: Unless there's a good reason I'm missing, function names should not be capitalised, and they should be snake_cased.

**Review by hillalex — APPROVED** [review]

**Comment by hillalex** [comment]
Thanks for these! The name `sql_equal` doesn't really make sense to me though, I guess that function should really be called `equal_or_null` or some variation thereof?

**Pull request merged**